src/platform_impl/windows/event_loop.rs

```diff
@@ -39,24 +39,7 @@ use windows_sys::Win32::UI::Input::Touch::{
 };
 use windows_sys::Win32::UI::Input::{RAWINPUT, RIM_TYPEKEYBOARD, RIM_TYPEMOUSE};
 use windows_sys::Win32::UI::WindowsAndMessaging::{
```

Comment:
I'm not sure what happened here. The requirement to use nightly rustfmt is kind of frustrating,